### DISCUSSION TITLE
End of 2024 update

### DISCUSSION THREAD

**Discussion by darobin** [darobin/dasl.ing#24] [General]
Dear all,

we just thought it would be a good idea to send you a post-call message about what we did as a result of the CID Congress conversation. Thanks a lot to everyone who joined the call, I wish all standards-adjacent discussions were this friendly and constructive. What a sweet community 💖💕

Here are a few things that happened as a result of our discussion (in no particular order):

- The specs have been thoroughly edited, and moved into separate docs.
- BLAKE3 has been dropped from CIDs. There is now a separate BDASL spec for that specific use case, which we expect not to see used on the web at large for now.
- dCBOR42 has been specified. Right now it supports floats. I know some of you want floats to die — we can keep debating it. I'm not religious about it either way, my thinking it more that it can be simpler to accept more complexity that is already supported off the shelf than to subset further.
- CAR has been specified. After looking at what people tend to support and asking around, this is just CAR v1. I heard very little interest in v2.
- The intro is a lot less about IPFS and the specs assume no IPFS knowledge.
- We haven't exactly deployed a massive outreach machine about DASL but we're already seeing some pick up here and there. Let us know when you spot it in the wild!
- We're looking into an IETF submission in 2025, at least of the CIDs and CAR, perhaps of dCBOR42 as a profile (but dCBOR already exists and tag 42 is already registered). An updated registration for tag 42 might be called for.

Other plans for 2025 include a test suite and the metadata format that people have been asking about (MASL). Let us know if you want other things, we endeavour to continue purveying the world's smallest specs.

If you know of an implementation that's on its way to supporting DASL but isn't done or needs support, talk to us! We have funding for implementations. We're also interested in work that makes it simpler to use DASL from broader IPFS implementation, e.g. by adding a profile parameter or a simpler interface.

Enjoy the end of 2024, we'll see you next year for more opinionated subsetting!

— Mosh & Bumblefudge & Robin

**Comment by bumblefudge** [comment]
Masl tov!
